chef/cookbooks/pacemaker/attributes/default.rb

```diff
@@ -15,15 +15,19 @@
 
 case node.platform
 when 'suse'
-  default[:pacemaker][:platform][:packages] = %w(pacemaker crmsh)
+  # Install xorg-x11-xauth by default so that 'crm configure graph'
+  # (which uses dotty) will show a nice configuration graph over ssh
+  # with X11 forwarding.
+  default[:pacemaker][:platform][:packages] = %w(
+    pacemaker crmsh xorg-x11-xauth
```

Comment:
If so, we could instead change the `hb_gui_hint` text to indicate that you need to set that option to yes in order for `crm configure graph` to work.